### DISCUSSION TITLE
Waiting for a new version

### DISCUSSION THREAD

**Discussion by fintmu** [hassio-addons/app-nut#242] [Q&A] [answered]
Hi,

I just got Legrand Keor PDU 800 UPS and noticed that support for it in NUT has been recently added. However this add-on still relies on UPS driver controller version 2.7.4 and newest release is v2.8.0 at the moment (released on April). Is new updated version coming soon from this add-on?

This add-on has been very nice since it simplifies basic NUT Server setup a lot, so thank you very much of your work on this.

**Comment by frenck** [comment, marked as answer]
The add-on runs on the latest version available to it.

**Reply by fintmu** [reply]
From the add-on (both official and edge versions):
![image](https://user-images.githubusercontent.com/56997162/182779276-425bbb73-9b72-4b39-983e-19dd58cb851d.png)

Latest version:
https://github.com/networkupstools/nut/releases/tag/v2.8.0

**Reply by frenck** [reply]
:( I'm so tired of this message and answer over and over again (not your fault!)

<https://github.com/hassio-addons/addon-nut/blob/main/nut/Dockerfile#L10>

The latest version available on Debian: <https://packages.debian.org/bullseye/nut>

Please... point me to the newer version available on Debian...

**Reply by fintmu** [reply]
Oh sorry, I should have dig this deeper... so have to wait new version on Debian, I understand. Thanks!
